### PR TITLE
docs(srv): restructure jobs and containers creation pages MTA-5185

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: serverless/jobs/how-to/create-job.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+tags: deploy create job scaleway-registry docker image external registry public
+dates:
+  validation: 2024-05-16
+  posted: 2023-11-09
+categories:
+  - serverless
+  - jobs
+---
+
+This page shows you how to create Serverless Jbs using the [Scaleway console](https://console.scaleway.com/).
```
Comment (RoRoJ) [inline serverless/jobs/how-to/create-job.mdx:17]:
```suggestion
This page shows you how to create Serverless Jobs using the [Scaleway console](https://console.scaleway.com/).
```

**Review by RoRoJ — APPROVED** [review]

File: serverless/containers/how-to/deploy-container.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- [Created a Container Registry namespace](/containers/container-registry/how-to/create-namespace/) and [pushed a container image](/containers/container-registry/how-to/push-images/) to it
+
+## Deploy from the Scaleway Container Registry
+
+1. Click **Containers** in the **Serverless** section of the side menu. The containers page displays.
+2. Click the relevant container's namespace.
+3. Click **Deploy container**. The container creation wizard displays.
+4. Complete the following steps in the wizard:
+    - Select the **Scaleway** Container Registry.
+    - Choose an [image](/serverless/containers/concepts/#image) from your Container Registry. Select the required Container Registry namespace from the drop-down list, and then select the container and tag.
```
Comment (nerda-codes) [inline serverless/containers/how-to/deploy-container.mdx:35]:
```suggestion
    - Choose an [image](/serverless/containers/concepts/#image) from your Container Registry. 
    - Select the required Container Registry namespace from the drop-down list, and then select the container and tag.
```

File: serverless/containers/how-to/deploy-container.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    - Choose the [port](/serverless/containers/concepts/#port) your container is listening on. We recommend configuring your container to listen on the `$PORT` environment variable.
+    - Choose a **name** for your container and, optionally, a **description**. The name must only contain alphanumeric characters and dashes.
+    - Choose the **resources** to be allocated to your container at runtime. These define the performance characteristics of your container.
+        <Message type="note">
+        Available memory depends on allocated vCPU, and maximum ephemeral storage value depends on allocated memory.
```
Comment (ldecarvalho-doc) [inline serverless/containers/how-to/deploy-container.mdx:40]:
```suggestion
        Available memory depends on allocated vCPU, and the maximum ephemeral storage value depends on allocated memory.
```

File: serverless/containers/how-to/deploy-container.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        </Message>
+    - Set your [scaling](/serverless/containers/concepts/#scaling) preferences, or leave them at default values. The Scaleway platform automatically scales the number of available instances of your container to match the incoming load, depending on the settings you define here.
+    - Click **Advanced options** to define any [environment variables](/serverless/containers/concepts/#environment-variables) you want to inject into your container. For each environment variable, click **+Add variable** and enter the key/value pair.
+    - Add [secrets](/serverless/containers/concepts/#secrets) for your container. Secrets are environment variables which are injected into your container, but the values are not retained or displayed by Scaleway after initial validation.
```
Comment (ldecarvalho-doc) [inline serverless/containers/how-to/deploy-container.mdx:44]:
```suggestion
    - Add [secrets](/serverless/containers/concepts/#secrets) for your container. Secrets are environment variables that are injected into your container, but the values are not retained or displayed by Scaleway after initial validation.
```

File: serverless/containers/how-to/deploy-container.mdx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
```
Comment (ldecarvalho-doc) [inline serverless/containers/how-to/deploy-container.mdx:79]:
```suggestion
    - Add [secrets](/serverless/containers/concepts/#secrets) for your container. Secrets are environment variables that are injected into your container, but the values are not retained or displayed by Scaleway after initial validation.
```

File: serverless/containers/how-to/create-a-containers-namespace.mdx [hunk 6]
```diff
@@ -29,6 +29,6 @@ This page shows you how to create a Serverless Containers namespace. [Namespaces
     - Enter any **environment variables** required for your namespace. Environment variables configured in a namespace will be available in all containers/apps within the same namespace. For each environment variable, click **+Add new variable** and enter the key / value pair.
     - Set secret environment variables (optional). **Secrets** are environment variables which are injected into your container and stored securely, but not displayed in the console after initial validation. Add a **key** and a **value**.
```
Comment (ldecarvalho-doc) [inline serverless/containers/how-to/create-a-containers-namespace.mdx:30]:
```suggestion
    - Set secret environment variables (optional). **Secrets** are environment variables that are injected into your container and stored securely, but not displayed in the console after initial validation. Add a **key** and a **value**.
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: serverless/jobs/how-to/create-job.mdx [hunk 7]
```diff
@@ -0,0 +1,101 @@
+---
+meta:
+  title: How to create a job
+  description: Create jobs using images from the Scaleway Container Registry or from external public registries.
+content:
+  h1: How to create a job
+  paragraph: Create jobs using images from the Scaleway Container Registry or from external public registries.
+tags: deploy create job scaleway-registry docker image external registry public
+dates:
+  validation: 2024-05-16
```
Comment (nerda-codes) [inline serverless/jobs/how-to/create-job.mdx:10]: maybe change the validation and posted dates

**Pull request merged**